Mortgage-loan-calculator/src/app/admin-login/services/storage.service.ts

```diff
@@ -1,47 +0,0 @@
-import { Injectable } from '@angular/core';
-
-const TOKEN_KEY = 'auth-token';
-const USER_KEY = 'auth-user';
-
-@Injectable({
-  providedIn: 'root'
-})
-export class StorageService {
-  constructor() {}
-
-  clean(): void {
-    window.sessionStorage.clear();
```

Comment:
When possible try to avoid using globals (such as window). In this case new InjectionToken should be created (`WINDOW`) and its value provided from global once. Later all access to `window` should be done through injected `WINDOW`.